frontend/styles/Variables.module.scss

```
@@ -20,10 +20,25 @@ $grey-lightest-color: #F1F1F1;
 $success-bg-color: #D4F4D2;
 $success-text-color: #4CAF50;
 $danger-color: #F00;
+// = color-mix(in srgb, $danger-color 8%, white), precomputed -- this file is plain SCSS vars,
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)